neqo-transport/src/connection/tests/zerortt.rs

```diff
@@ -277,17 +277,15 @@ fn zero_rtt_loss_accepted() {
         let mut now = now();
         let client_stream_id = client.stream_create(StreamType::UniDi).unwrap();
         client.stream_send(client_stream_id, &[1, 2, 3]).unwrap();
-        let ci = client.process(None, now);
+        let mut ci = client.process(None, now);
```

Comment:
I removed a layer of nesting here and ensured that you passed in the correct packet (the last, not the first).

Reply:
Thanks! I have this locally but didn't push yet.